processing/src/main/java/org/apache/druid/segment/filter/Filters.java

```diff
@@ -651,4 +652,17 @@ private static void generateAllCombinations(
       generateAllCombinations(result, andList.subList(1, andList.size()), nonAndList);
     }
   }
+
+  public static boolean shouldUseIndex(Filter filter, BitmapIndexSelector indexSelector, FilterTuning filterTuning)
```

Comment:
Could use a bit of javadocs as to its purpose ("standard" implementation of Filter.shouldUseIndex) and how it works (returns true if the filter supports indexes and if all underlying columns match the thresholds in `filterTuning`). Also that `filterTuning` is nullable.

Reply:
Thanks, this looks good. Marking the filterTuning parameter `@Nullable` would be helpful too.